cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -151,7 +151,7 @@ install_docker(){
 			sudo -E apt-get -y install "${pkg_name}=${docker_version_full}"
 		elif [[ "$ID" =~ ^opensuse.*$ ]] || [ "$ID" == "sles" ]; then
 			sudo zypper removelock docker
-			sudo zypper -n  install 'docker<19.03'
+			sudo zypper -n  install docker
```

Comment:
By the moment we do this, we lose control on what's the specific version of docker being installed. @GabyCT, IIRC, we're testing using a specific version for a reason, do you remember what's the reasoning?. Are we fine bumping it there?

@rhafer, out of curiosity, what's the reason 

All in all, I'm in favour of switching to newer versions of docker, it's a step forward.

Reply:
> @rhafer, out of curiosity, what's the reason

@fidencio The main reason is that recent SLES and openSUSE releases (that is SLES 15 SP2, and Leap 15.2) only ship with a newer docker version. If you think it's better I could pin it to `19.03`. However that would still be different to what's used on the other distros.

AFAIK one of the reason to stick with a old version of docker was that the Debian/Ubuntu packages had devicemapper support removed  with 18.09. However, it seems that that change has been reverted for 19.03. So it might indeed be possible try 19.03 for other distros as well. But I guess @GabyCT has some more insight into that.

Reply:
yes we are on that version because of the devicemapper support

Reply:
@GabyCT, what are your feelings about moving to 19.03, considering the changes were reverted for 19.03?

Reply:
for me it is ok to move on to 19.03 but I think the best is to do across all the other distros as we have documentation and so on where we are specifying a specific docker version
/cc @chavafg

Reply:
https://github.com/kata-containers/documentation/blob/master/install/docker/sles-docker-install.md

Reply:
I'm fine with it, but as @GabyCT mentions, it would be good to have the same docker version across all distros. Would also be good to know if this (currently skipped) test can pass now with the update: https://github.com/kata-containers/tests/blob/master/integration/stability/hypervisor_stability_kill_test.sh#L28-L29.

Reply:
> Would also be good to know if this (currently skipped) test can pass now with the update: https://github.com/kata-containers/tests/blob/master/integration/stability/hypervisor_stability_kill_test.sh#L28-L29.

Hm, that is still failing with newer docker releases :(.  However I think I have a rough understanding why that is. I'll followup in https://github.com/kata-containers/runtime/issues/2611